### ISSUE TITLE
[a11y] Cannot open more than 1 dialog in a tree

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1603]
On `edge` you can open multiple dialogs. This helps compare records before synonymizing or merging. In `a11y`, this feature is absent. One dialog obscures the ability to navigate the tree.

**Comment by maxxxxxdlp** [comment]
Begun fixing it by making dialog window non-modal.
The window displays the currently focused node.
Focusing a different node updates the dialog window
@grantfitzsimmons Can you please test that this did not introduce any bugs.
Having more that one dialog open was buggy in the previous release. Will try to fix them and add this feature in the future
